ormolu-live/cbits/init.c

```diff
@@ -0,0 +1,12 @@
+#include "Main_stub.h"
+#include <Rts.h>
+
+__attribute__((export_name("wizer.initialize"))) void __wizer_initialize(void) {
+  char *args[] = {"ormolu-live.wasm", "+RTS", "--nonmoving-gc", "-RTS", NULL};
+  int argc = (int)(sizeof(args) / sizeof(args[0]));
```

Comment:
```suggestion
  int argc = sizeof(args) / sizeof(args[0]) - 1;
```

The trailing `NULL` doesn't count into the `argc`.

Reply:
LOL smh